groups/data-lake/variables.tf

```diff
@@ -1,22 +1,8 @@
-variable "region" {}
-
-variable "database_password" {
-  type        = string
-  description = "The Redshift database password"
-}
-
-variable "database_username" {
-  type        = string
-  description = "The Redshift database username"
-}
-
-variable "mongo_export_db_url" {
-  type        = string
-  description = "The MongoDB URL for the export Lambda"
+variable "aws_profile" {
+  default = "development-eu-west-2"
+  type = string
```

Comment:
Can we document all the variables for completeness? Here's what we use in other projects:

```
description = "The AWS profile name; used as a prefix for Vault secrets"
```